saleor/graphql/order/tests/mutations/test_order_line_update.py

```diff
@@ -417,3 +419,130 @@ def test_order_line_update_quantity_gift(
     assert len(errors) == 1
     assert errors[0]["field"] == "id"
     assert errors[0]["code"] == OrderErrorCode.NON_EDITABLE_GIFT_LINE.name
+
+
+ORDER_LINE_UPDATE_WITH_DISCOUNTS = """
```

Comment:
I will suggest to use and extend `ORDER_LINE_UPDATE_MUTATION` query instead of introducing a new one